prescriptions-refresh-job/base/argo-workflows/prescriptions-refresh-image-analysis.yaml

```diff
@@ -1,110 +1,167 @@
-              secretKeyRef:
-                name: kebechet
-                key: GITHUB_APP_ID
-          - name: THOTH_PRESCRIPTIONS_REFRESH_GITHUB_LABELS
-            value: bot
-          - name: THOTH_PRESCRIPTIONS_REFRESH_REPO
-            value: git@github.com:thoth-station/prescriptions.git
-          - name: THOTH_PRESCRIPTIONS_REFRESH_DEBUG
-            value: "1"
-          - name: PROMETHEUS_PUSHGATEWAY_HOST
-            valueFrom:
-              configMapKeyRef:
-                key: pushgateway-host
-                name: prometheus
-          - name: PROMETHEUS_PUSHGATEWAY_PORT
-            valueFrom:
-              configMapKeyRef:
-                key: pushgateway-port
-                name: prometheus
-        resources:
-          limits:
-            cpu: 250m
-            memory: 256Mi
-          requests:
-            cpu: 250m
-            memory: 256Mi
-        volumeMounts:
-          - name: ssh-config
-            mountPath: /opt/app-root/src/.ssh
-          - name: github-app-privatekey
-            mountPath: /opt/app-root/src/.github/
-            readOnly: true
+        - name: duration_seconds_histogram
+          help: Duration of workflow when succeeded
+          when: "{{workflow.status}} == Succeeded"
+          labels:
+            - key: name
+              value: prescriptions-refresh-image-analysis
+          histogram:
+            buckets:
+              - 5
+              - 10
+              - 30
+              - 60
+              - 120
+              - 180
+              - 300
+              - 600
+              - 900
+            value: "{{workflow.duration}}"
+    volumes:
+      - name: ssh-config
+        secret:
+          secretName: kebechet
+          items:
+            - key: ssh-privatekey
+              path: id_rsa
+              mode: 0600
+      - name: github-app-privatekey
+        secret:
+          secretName: kebechet
+          items:
+          - key: KEBBHUT_GITHUB_PRIVATE_KEY
+            path: github-privatekey
+    entrypoint: image-analysis
+    templates:
+      - name: image-analysis
+        resubmitPendingPods: true
+        container:
+          name: image-analysis
+          image: prescriptions-refresh-job
+          env:
+            - name: THOTH_PRESCRIPTIONS_DRY_RUN
+              value: 1
+            - name: SENTRY_DSN
+              valueFrom:
+                secretKeyRef:
+                  name: thoth
+                  key: sentry-dsn
+            - name: GIT_SSH_COMMAND
+              # Needed to keep weak host verification for git clone.
+              value: "ssh -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no"
+            - name: THOTH_DEPLOYMENT_NAME
+              valueFrom:
+                configMapKeyRef:
+                  name: thoth
+                  key: deployment-name
+            - name: THOTH_USER_API_HOST
+              valueFrom:
+                configMapKeyRef:
+                  name: prescription-refresh-job
```

Comment:
```suggestion
                  name: prescriptions-refresh-job
```

Reply:
Fixed in https://github.com/thoth-station/thoth-application/pull/2312/commits/eee94e2dc151a5a32ecf5b897ca49259e99ff63b